sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpEvent.kt

```diff
@@ -0,0 +1,158 @@
+package io.sentry.android.okhttp
+
+import io.sentry.Breadcrumb
+import io.sentry.Hint
+import io.sentry.IHub
+import io.sentry.ISpan
+import io.sentry.SpanStatus
+import io.sentry.TypeCheckHint
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.CONNECTION_EVENT
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.CONNECT_EVENT
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.REQUEST_BODY_EVENT
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.REQUEST_HEADERS_EVENT
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.RESPONSE_BODY_EVENT
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.RESPONSE_HEADERS_EVENT
+import io.sentry.android.okhttp.SentryOkHttpEventListener.Companion.SECURE_CONNECT_EVENT
+import io.sentry.util.UrlUtils
+import okhttp3.Call
+import okhttp3.Response
+import java.net.URL
+
+private val uuidRegex by lazy { Regex("[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}") }
+
+internal class SentryOkHttpEvent(private val hub: IHub, private val call: Call) {
```

Comment:
nit: do we still have to hold reference to the `call` variable? We're only using it in the `init` block, so I think it's not necessary
```suggestion
internal class SentryOkHttpEvent(private val hub: IHub, call: Call) {
```